lib/elliptic/ec/key.js

```diff
@@ -51,13 +52,7 @@ KeyPair.prototype.validate = function validate() {
   return { result: true, reason: null };
 };
 
-KeyPair.prototype.getPublic = function getPublic(compact, enc) {
-  // compact is optional argument
-  if (typeof compact === 'string') {
-    enc = compact;
-    compact = null;
-  }
-
+KeyPair.prototype.getPublic = function getPublic(enc, compact) {
```

Comment:
This is going to be a major semver change.

Reply:
Do you know when you'd like to release v7.0.0? I can think of some other possibly breaking cleanups that might be worthwhile to do.

For instance, I think it would make sense to remove dependency on `hmac-drbg` and `brorand` and instead pass a callback function that returns randomness. This one is definitely breaking, but there are some other things that I can't quite remember off the top of my head.

Reply:
Don't know about this one. This breaks node.js convention: `enc` is typically the last argument. It's also going to be a huge pain replacing all the code which relies on the old parameter order.

Agree with removing the brorand dependency though.

Reply:
I think consistency is more important here. Also, I've done the work of replacing the code which relies on the old order in this project (which was minimal), and other projects can easily fix it whenever they update the vendored version of elliptic.